packages/react-liveness/src/components/FaceLivenessDetector/utils/helpers.ts

```diff
@@ -8,5 +8,4 @@ export const STATIC_VIDEO_CONSTRAINTS: MediaTrackConstraints = {
     ideal: 480,
   },
   frameRate: { min: 15, ideal: 30, max: 30 },
-  facingMode: 'user',
```

Comment:
Just to confirm, this is no longer needed because on no light challenge we only show the dropdown on desktop, and on desktop the facing mode is not relevant?

Reply:
Exactly, desktop cameras do not have a facingMode value